performance/tests/sandstone/ListItemsTests.js

```diff
@@ -34,12 +36,38 @@ const listItemTests = (componentName, dataSize) => describe(componentName, () =>
 		});
 	});
 
+	describe('ScrollButton', () => {
+		it('scrolls down with translate scrollMode', async () => {
+			const pageURLTranslate = pageURL + '?scrollMode=translate';
```

Comment:
scrollMode is shown as undefined when dataSize is set. You should use "&" at the beggining in that case